app/scripts/controllers/npSpecializations.js

```diff
@@ -347,9 +362,13 @@
       result.name = name;
 
       // additional properties
-      // result.okrId = okrId || null;
-      // result.npQualificationId = npQualificationId || null;
-      // result.actuality = actuality || null;
+      result.okrId = !okrId ? null : okrId;
+      result.actuality = !actuality ? false : actuality;
+      result.protocolNumber = !protocolNumber ? null : okrId;
+      // temp solution before backend fix problem with null protocolDate property
```

Comment:
It is additional (not required) property of `NpViewModel` so client might fill this property or might not fill.